pkg/test/expectations/expectations.go

```diff
@@ -734,3 +738,10 @@ func ConsistentlyExpectNotTerminating(ctx context.Context, c client.Client, objs
 		}
 	}, time.Second).Should(Succeed())
 }
+
+func ExpectNewTestingQueue(c client.Client, recorder events.Recorder, cluster *state.Cluster, clock clockiface.Clock, provisioner *provisioning.Provisioner) *orchestration.Queue {
```

Comment:
Do we need this change? Also, I would say that this is a constructor and scoped specifically to disruption so keeping it where it was makes more sense IMO